docs/lib/push-notifications/fragments/js/getting-started.md

```diff
@@ -174,17 +174,10 @@ Push Notifications category is integrated with [AWS Amplify Analytics category](
     $ react-native init myapp
```

Comment:
Beyond what you changed, but this is decidedly not JavaScript :)

Can we make this `bash`, make it readable (line breaks), and remove the prompts so a user could copy and paste it?

Reply:
updated